docs/schemas/Donor-Metadata/donor-metadata.csv

```diff
@@ -0,0 +1,145 @@
+education_years_number,add3f7fe-ae60-4cc1-bb2e-72bbbe9a1a7d,Education Years Number,education_years_number,,,,,0-30,,,CDE: C00015 CDE defines it as 5 and up. ,,,numeric,1,6/6/23,General Subject Fields,Donor Metadata,,0,30,year
+family_income_range,6cd785e2-7333-43fe-8c8c-c19532e96ea6,Family Income Range,family_income_range,,,,,[Under $15000; $15000 to $24999; $25000 to $34999; $35000 to $49999; $50000 to $74999; $75000 to $99999; $100000 and over; Refused; Unknown],[Under $15000; $15000 to $24999; $25000 to $34999; $35000 to $49999; $50000 to $74999; $75000 to $99999; $100000 and over; Refused; Unknown],[Under $15000; $15000 to $24999; $25000 to $34999; $35000 to $49999; $50000 to $74999; $75000 to $99999; $100000 and over; Refused; Unknown],,,[Income Range CV],exclusive_categorical,1,6/6/23,General Subject Fields,Donor Metadata,,,,
+informant_questionnaire_completed,e8b9666f-a112-456f-aa5a-69a101c3866f,Informant Questionnaire Completed,informant_questionnaire_completed,,,,,[Yes; No],,[Yes; No],,Boolean?,,exclusive_categorical,1,6/6/23,General Subject Fields,Donor Metadata,,,,
+informant_interview_performed,274b838b-74c6-46ec-808a-0c080ddeedd9,Informant Interview Performed,informant_interview_performed,,,,,[Yes; No],,[Yes; No],,Boolean?,,exclusive_categorical,1,6/6/23,General Subject Fields,Donor Metadata,,,,
+informant_relationship,836c19fe-b735-4bf7-bee5-3213122b571e,Informant Relationship,informant_relationship,,,,,,,[Child; Friend/Neighbor; Parent; Professional Caregiver; Self; Sibling; Spouse/Partner; Teammate; Other Relative (specify); Other (specify)],,I'm shocked that there are no informant relation terms in NIH CDE repository. ,[Informant Relation CV],exclusive_categorical,1,6/6/23,General Subject Fields,Donor Metadata,,,,
+informant_relationship_specify,289bcbb8-5177-4c65-b0af-159fd65b69aa,Informant Relationship Specify,informant_relationship_specify,,,,,,,,,,,text,1,6/6/23,General Subject Fields,Donor Metadata,,,,
+handedness,82a4087c-d0e3-48ca-8bea-a81eefc683a0,Handedness,handedness,,,,,,,[Left; Right; Either],,,[Handedness CV],exclusive_categorical,1,6/6/23,General Subject Fields,Donor Metadata,,,,
+medical_records_available,ee2c9223-f7d1-4c64-a1da-cd7408309955,Medical Records Available,medical_records_available,,,,,,,[Yes; No],,Boolean?,,exclusive_categorical,1,6/6/23,General Subject Fields,Donor Metadata,,,,
+medical_records_reviewed,6139b159-c36c-4501-9176-fe2741e3d448,Medical Records Reviewed,medical_records_reviewed,,,,,,,[Yes; No],,Boolean?,,exclusive_categorical,1,6/6/23,General Subject Fields,Donor Metadata,,,,
+hemisphere,68b7a28b-d695-4ec9-800b-6a27ca206081,Hemisphere,hemisphere,,,,,[Left; Right; Unknown],,[Left; Right; Both; Unknown],"What is significance of ""both"" in SP?",,,exclusive_categorical,1,6/6/23,General Specimen Data,Donor Metadata,,,,

```

Comment:
This field seems redundant, since left and right_hemisphere_preparation fields both have 'hemisphere not available' as part of their value set